spec/02-access_spec.lua

```diff
@@ -463,7 +463,7 @@ do
 
       assert.res_status(200, res)
       assert.same("Miss", res.headers["X-Cache-Status"])
-      local cache_key = res.headers["X-Cache-Key"]
+      --local cache_key = res.headers["X-Cache-Key"]
```

Comment:
```suggestion
```

Reply:
I commented it because it relates to other already commented out lines of code. Do you think we should remove them too, e.g. the lines exactly below this?